roles/confluent.variables/vars/main.yml

```diff
@@ -49,13 +49,15 @@ zookeeper_properties:
       ssl.quorum.trustStore.location: "{{zookeeper_truststore_path}}"
       ssl.quorum.trustStore.password: "{{zookeeper_truststore_storepass}}"
       ssl.clientAuth: "{{ 'need' if zookeeper_ssl_mutual_auth_enabled|bool else 'none' }}"
-  ssl_fips:
-    enabled: "{{ zookeeper_ssl_enabled and pkcs12_enabled }}"
-    properties:
-      ssl.keymanager.algorithm: PKIX
-      ssl.trustmanager.algorithm: PKIX
-      ssl.quorum.keymanager.algorithm: PKIX
-      ssl.quorum.trustmanager.algorithm: PKIX
+  # TODO need to understand why these variables were necessary, its not bc pkcs12 keystores were used bc pkcs12 keystores work without these Variables
```

Comment:
@prestonmcgowan This PR is a still a work in progress, but the plan is to use type PKCS12 keystores  (created without the bouncy castle jar) for ZK and I've found they do not need these props: `ssl.keymanager.algorithm: PKIX` 

Trying to figure out why they were needed in the first place. Any thoughts?

Reply:
@jeffhuang26 says using PKIX is a good idea and is valid for PKCS12 keystore type, so should leave this code in